sklearn/preprocessing/_encoders.py

```diff
@@ -699,6 +699,26 @@ class OrdinalEncoder(_BaseEncoder):
     dtype : number type, default np.float64
         Desired dtype of output.
 
+    missing_value : object type or a list of object types, default "nan"
```

Comment:
Called missing_values (plural) elsewhere. Let's be consistent

Reply:
We can't really accept a string "nan" which is ambiguous when X can be strings

Reply:
And I'd rather support NaN by default for now and not make it user-configurable

Reply:
> And I'd rather support NaN by default for now and not make it user-configurable

\+1 
Also, if we keep it as an option, I would not make it a list (eg in SimpleImputer it is also only a single value, we should be consistent with that IMO)

Reply:
> We can't really accept a string "nan" which is ambiguous when X can be strings

@maxcopeland to give some context: in imputers currently it is a string "nan", but in master (and in soon to be released 0.20), this was changed to actually be `np.nan`.